alicloud/table_alicloud_compute_zone.go

```diff
@@ -0,0 +1,166 @@
+package alicloud
+
+import (
+	"context"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudComputeZone(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_compute_zone",
+		Description: "",
+		List: &plugin.ListConfig{
+			ParentHydrate: listComputeRegions,
+			Hydrate:       listComputeZones,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
+			// Top columns
+			{
+				Name:      "zone_no",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("ZoneNo"),
```

Comment:
Add column description

Reply:
zone_no will be removed , it does not fetch any value , present in API but not in documentation